expo/scheduler/models.py

```diff
@@ -449,6 +449,11 @@ def get_conflicts(self, new_event):
                 conflicts += [event]
         return conflicts
 
+    def volunteer_events(self, conference):
```

Comment:
this seems similar but different from get_bookings.  Could you combine them?  Filtering by conference (if present) sounds great.  And if you're getting only the volunteer events, then they should be only for the role="Volunteer" - I'd prefer to have fewer functions here, so that if we ever have a bug to fix, or a logic change, it's in common code.